src/marqo/tensor_search/health.py

```diff
@@ -48,6 +146,9 @@ def get_marqo_os_status(config: Config, index_name: Optional[str] = None) -> Uni
     else:
         marqo_os_status = HealthStatuses.red
 
```

Comment:
Nit: In the doc string of this method we can record the different requests to OpenSearch that will be made. This will help us be conscious of the number of OpenSearch requests.

Reply:
done

Reply:
Do you need to push changes? I don't see it updated yet